core/src/main/java/org/bitcoinj/core/Context.java

```diff
@@ -77,22 +78,23 @@ public Context(NetworkParameters params) {
      * @param feePerKb The default fee per 1000 virtual bytes of transaction data to pay when completing transactions. For details, see {@link SendRequest#feePerKb}.
      * @param ensureMinRequiredFee Whether to ensure the minimum required fee by default when completing transactions. For details, see {@link SendRequest#ensureMinRequiredFee}.
      */
-    public Context(int eventHorizon, Coin feePerKb, boolean ensureMinRequiredFee) {
+    public Context(int eventHorizon, Coin feePerKb, boolean ensureMinRequiredFee, boolean relaxProofOfWork) {
```

Comment:
`relaxProofOfWork` should be added to JavaDoc

Reply:
Done.